api/src/main/resources/workbench.yaml

```diff
@@ -380,7 +380,42 @@ paths:
           type: string
           required: true
           description: ID of the workspace containing the cohort definition
-
+  /api/v1/workspaces/review:
```

Comment:
We may want this method to take in a "size" / "maxWorkspaces" / "count" parameter as input, to limit response sizes, and change WorkspaceListResponse so that it can optionally contain a total count.

If you don't want to bother with any of this right now, that's cool -- but file a JIRA ticket to make sure we don't forget later.

Reply:
It could also be that requesting a review of research purpose is sufficiently rare, and our number of researchers sufficiently small, that this will never be an issue. :)